src/App.js

```diff
@@ -1,24 +1,34 @@
-import logo from './logo.svg';
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Help from './pages/Help';
+import Map from './pages/Map';
+import List from './pages/List';
+import Settings from './pages/Settings';
+import Nav from './components/Nav';
+
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
```

Comment:
Why do we need the div wrapped around the routes here?

Reply:
We just copied from the docs here! https://reactrouter.com/web/guides/quick-start

Reply:
@greffly said she tried it locally without the div & it worked. So I'll delete it before merging :)